categorize_adjectives/keywords.py

```diff
@@ -0,0 +1,82 @@
+"""Categorize Adjectives by looking for keywords in definition"""
+import os
+from nltk.corpus import wordnet as wn
+
+# Folder where generated files are stored
+generated_file = os.path.join(os.getcwd(), '..', 'generated_files/')
+
+# Fetching all adjective synsets from Wordnet
+adjectives = wn.all_synsets('a')
+
+# Semantic Categories for adjectives
+categories = {'Perceptional' : [], 'Spatial' : [], 'Time' : [], 'Motion' : [], 'Substance' : [],
+              'Weather' : [], 'Body' : [], 'Mood' : [], 'Spirit' : [], 'Behavior' : [],
+              'Social' : [], 'Quantity' : [], 'Relational' : [], 'General' : [], 'Others' : []}
+
+# Keywords used to lookout for adjectives
```

Comment:
Can you please add a link to the paper here from where we picked up these categories?